docs/2021-02-08-metabase.md

```diff
@@ -144,7 +144,7 @@ from the analytics schema.
 Create a new Google API token for use with Metabase. The details
 tend to change over time, but this worked at the time of writing:
 
-* Visit https://console.cloud.google.com/ and make sure you are in the **getyourrefund-304319** project.
+* Visit https://console.cloud.google.com/ and make sure you are in the **getyourrefund** project.
```

Comment:
Heh. Lovely. Are we all part of this project, or should we make sure that we are?